packages/auth/src/features/authenticate/handler.test.ts

```diff
@@ -54,7 +55,7 @@ describe('authenticate handler receives a request', () => {
     it('returns 403', async () => {
       fetch.mockResponse(
         JSON.stringify({
-          userId: '1',
+          id: '1',
```

Comment:
This must have been a mistake in the tests. `/verifyPassword` is what is being mocked here but it never did return a `userId` field https://github.com/opencrvs/opencrvs-core/blob/754ae62bd38e3cb533961e2bc866f119f7d67c6a/packages/user-mgnt/src/features/verifyPassword/handler.ts#L92